_inc/client/components/jetpack-termination-dialog/survey.jsx

```diff
@@ -0,0 +1,64 @@
+/**
+ * External dependencies
+ */
+import { translate as __ } from 'i18n-calypso';
+import Card from 'components/card';
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import MultiChoiceQuestion from 'components/multiple-choice-question';
+
+// these answers should line up exactly with the options in Calypso
```

Comment:
While this works right now, I'm afraid it's going to be hard to ensure the 2 stay in sync, unless:
- We had comments here and in the Calypso counterpart asking that when updating one, please update there as well, with a link to the counterpart.
- We pull this content from an API endpoint?

Reply:
endpoint would be lovely, but out of my scope for this week 😞 I will clarify the assumptions here and link to the survey in calypso.